internal/lm/nvml.go

```diff
@@ -218,6 +224,45 @@ func newGPUModeLabeler(devices []resource.Device) (Labeler, error) {
 	return labels, nil
 }
 
+func newImexDomainLabeler(devices []resource.Device) (Labeler, error) {
+	var commonClusterUUID, commonCliqueID string
```

Comment:
nit: could we separate these lines? It makes future diffs easier.

Reply:
PR now on new direction